index.js

```diff
@@ -279,43 +279,44 @@ const fallback = {
 };
 
 if (platform === 'linux') {
-	// The main one doesn't look that good on Ubuntu.
-	main.circleQuestionMark = '?';
-	main.questionMarkPrefix = '?';
+	// The main symbols for those do not look that good on Ubuntu.
+	mainSymbols.circleQuestionMark = '?';
+	mainSymbols.questionMarkPrefix = '?';
 }
 
 // TODO: Use https://github.com/sindresorhus/is-unicode-supported when targeting Node.js 10.
-const figures = platform === 'win32' ? fallback : main;
+const shouldUseWindows = platform === 'win32';
+const figures = shouldUseWindows ? windowsSymbols : mainSymbols;
 
-const isFallbackFigure = ([key, mainValue]) => figures[key] !== mainValue;
-const getFigureRegExp = ([key, mainValue]) => [new RegExp(escapeStringRegexp(mainValue), 'g'), figures[key]];
+const isWindowsSymbol = ([key, mainSymbol]) => figures[key] !== mainSymbol;
+const getFigureRegExp = ([key, mainSymbol]) => [new RegExp(escapeStringRegexp(mainSymbol), 'g'), windowsSymbols[key]];
```

Comment:
`figures` is always evaluated to `windowsSymbols` here. This replaces it for clarity.